src/integrators/controllers.jl

```diff
@@ -580,7 +580,9 @@ function stepsize_controller!(integrator, alg::FBDF{max_order}) where max_order
         terk_tmp *= abs(dt^(k-2))
       else
         for i in 2:k-2
-          @.. @views terk_tmp += fd_weights[i,k-2] * u_history[:,i-1]
+          for j in eachindex(terk_tmp)
+            terk_tmp[j] += fd_weights[i,k-2] * u_history[j,i-1]
```

Comment:
Any particular reason to use loops here?

Reply:
No, just forgot it. I will carefully check git diff next time :(